src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
+  const half = 20;
+  const week = 50;
```

Comment:
What are `cost`, `half` and `week`, be more specific with your variable names
```suggestion
  const totalDailyCost = days * 40;
  const smallDiscount = 20;
  const bigDiscount  = 50;
```